opentelemetry-semantic-conventions/src/opentelemetry/semconv/resource/__init__.py

```diff
@@ -126,25 +131,67 @@ class ResourceAttributes:
     Name of the [deployment environment](https://en.wikipedia.org/wiki/Deployment_environment) (aka deployment tier).
     """
 
+    DEVICE_ID = "device.id"
+    """
+    A unique identifier representing the device.
+    Note: The device identifier MUST only be defined using the values outlined below. This value is not an advertising identifier and MUST NOT be used as such. On iOS (Swift or Objective-C), this value MUST be equal to the [vendor identifier](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor). On Android (Java or Kotlin), this value MUST be equal to the Firebase Installation ID or a globally unique UUID which is persisted across sessions in your application. More information can be found [here](https://developer.android.com/training/articles/user-data-ids) on best practices and exact implementation details. Caution should be taken when storing personal data or anything which can identify a user. GDPR and data protection laws may apply, ensure you do your own due diligence.
```

Comment:
Can we update the template to reflow long text? https://jinja.palletsprojects.com/en/2.11.x/templates/#wordwrap

Reply:
Looks like if I update `generate.sh` like this:

```
-    {{attribute.brief | to_doc_brief}}.
+    {{attribute.brief | to_doc_brief | wordwrap}}.
```

it doesn't do the tabs properly 😕 

```
    DEVICE_ID = "device.id"
    """
    A unique identifier representing the device.
    Note: The device identifier MUST only be defined using the values outlined below.
This value is not an advertising identifier and MUST NOT be used as such. On
iOS (Swift or Objective-C), this value MUST be equal to the [vendor identifier]
(https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierfor
vendor). On Android (Java or Kotlin), this value MUST be equal to the Firebase
Installation ID or a globally unique UUID which is persisted across sessions in
your application. More information can be found
[here](https://developer.android.com/training/articles/user-data-ids) on best
practices and exact implementation details. Caution should be taken when
storing personal data or anything which can identify a user. GDPR and data
protection laws may apply, ensure you do your own due diligence.
    """
```